chainer/train_imagenet.py

```diff
@@ -64,45 +91,38 @@ def train_loop():
     label = np.ndarray((args.batchsize), dtype=np.int32)
     label.fill(1)
     count = 0
+    timer = Timer()
     for i in range(niter):
         x = xp.asarray(data)
         y = xp.asarray(label)
         
         if args.arch == 'googlenet':
-            start = xp.cuda.Event()
-            end = xp.cuda.Event()
-            start.record()
+            timer.preprocess()
```

Comment:
This is OK for CPU, but to measure the elapsed time on GPU, it's not correct way. So could you split this file to different file to benchmark only CPU, like `train_imagenet_cpu.py`?